Src/FluentAssertions/TypeEnumerableExtensions.cs

```diff
@@ -77,5 +77,60 @@ public static IEnumerable<Type> ThatImplement<T>(this IEnumerable<Type> types)
         {
             return new TypeSelector(types).ThatImplement<T>();
         }
+
+        /// <summary>
+        /// Filters to only include types that are classes.
+        /// </summary>
+        public static IEnumerable<Type> ThatAreClasses(this IEnumerable<Type> types)
+        {
+            return new TypeSelector(types).ThatAreClasses();
+        }
+
+        /// <summary>
+        /// Filters to only include types that are not classes.
+        /// </summary>
+        public static IEnumerable<Type> ThatAreNotClasses(this IEnumerable<Type> types)
+        {
+            return new TypeSelector(types).ThatAreNotClasses();
+        }
+        /// <summary>
+        /// Filters to only include types that are static classes.
+        /// </summary>
+        public static IEnumerable<Type> ThatAreStaticClasses(this IEnumerable<Type> types)
+        {
+            return new TypeSelector(types).ThatAreStaticClasses();
```

Comment:
Should these just be named `ThatAre[Not]Static()`?
That's what the implementation seems to do.

Reply:
This is the place where I was also taken aback. 
What do you think, the name `That [Not] AreStatic` is not confusing, as if it were about static methods and properties?

Reply:
BTW, is a Type Extensions. I'll name it as you suppose.

Reply:
Done